app/controllers/subscriptions_controller.rb

```diff
@@ -10,6 +10,8 @@ def create
     else
       redirect_to root_path, error: subscription.errors.full_messages.to_sentence
     end
+  rescue ActiveRecord::RecordInvalid
```

Comment:
Instead of rescuing from the exception, we can avoid it all together. 
https://stackoverflow.com/a/27421738